### PR TITLE
Add Sentiment Analysis button for comments that reports to user how comments score based on Cloud API.

### PR DESCRIPTION
**Pull request by anooshrees**

### CODE REVIEW

File: portfolio/application_default_credentials.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "client_id": "764086051850-6qr4p6gpi6hn506pt8ejuq83di341hur.apps.googleusercontent.com",
```
Comment (yang-julian) [inline portfolio/application_default_credentials.json:2]: There are people that scrape github for api keys.  Is this key only usable on your particular project?

**Review by yang-julian — CHANGES_REQUESTED** [review]

**Comment by anooshrees** [comment]
Yep! Only works for the Anooshree-step-2020 project.

**Comment by anooshrees** [comment]
image

**Comment by anooshrees** [comment]
image

**Review by yang-julian — APPROVED** [review]

**Pull request merged**